CommonTools/ParticleFlow/python/EITopPAG_cff.py

```diff
@@ -175,3 +175,12 @@
     pfCombinedInclusiveSecondaryVertexV2BJetTagsEI
     )
 
+## customizations for the pp_on_AA eras                                                                                                                                                        
+from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+(pp_on_XeXe_2017 | pp_on_AA_2018).toModify(pfPileUpJMEEI,
+                                           Vertices = "offlinePrimaryVertices"
+                                           )
+(pp_on_XeXe_2017 | pp_on_AA_2018).toModify(pfJetsEI,
+                                           srcPVs = "offlinePrimaryVertices"
```

Comment:
please add goodOfflinePrimaryVertices to the front of the EIsequence.
This is a more appropriate fix.

This fix here is needed indirectly still in connection with the same problem triggered by removal of the pfNoPileUpJME mentioned earlier in https://github.com/cms-sw/cmssw/pull/24380/files#r218527492.
It seems like there was no check of running without PAT step in the "fix" PR #24655